release notes/v0.38.0.md

```diff
@@ -0,0 +1,275 @@
+
+The previous behavior can still be achieved by casting the `Object` to a `String`, as in `console.log(String({'foo': 'bar'}))`.  
+
+**Objects**
+
+```javascript
+export default function () {
+  var user = [{}]
+
+  console.log("the user", user) 
+  // before: the user [object Object]
+  // after: the user {"name":"Bob"}
+}
+```
+
+**arrays**
+
+```javascript
+export default function () {
+  var user = {name: "Bob"}
+
+  console.log([1, 2, "mytest", {user: "Bob"}]) 
+  // before: 1,2,mytest,[object Object]
+  // after: [1,2,"mytest",{"user":"Bob"}]
+}
+```
+
+### Disabling colors ([2410](https://github.com/grafana/k6/pull/2410))
+
+k6 output colors can now be disabled. Following the *NO_COLOR* standard, setting the `NO_COLOR` or `K6_NO_COLOR` environment variables allow you to disable k6's usage of ANSI color escape codes.
+
+```bash
+# No color output
+K6_NO_COLOR=true k6 run script.js
+
+# No color output
+NO_COLOR= k6 run script.js
+```
+
+### Deprecations
+
+* [2499](https://github.com/grafana/k6/pull/2499/files) removed support for the deprecated max VU's `max` **option**.
+*  dropped The deprecated `common.Bind` ([2488](https://github.com/grafana/k6/pull/2448)) and `common.BindToGlobal` ([2451](https://github.com/grafana/k6/pull/2451)) functions as well as the context-based (`common/context.go` [2488](https://github.com/grafana/k6/pull/2448)) utils have been removed. Those were left-over from the previous version of our JS module APIs and are unsupported as of **v0.38.0**.
+   
+## Enhancements and UX improvements
+
+### Stricter thresholds' evaluation before the execution starts ([#2330](https://github.com/grafana/k6/issues/2330))
+
+k6 **v0.37.0** already improved thresholds parsing by switching its underlying implementation from JavaScript to Go. k6 **v0.38.0** introduces two additional improvements:
+* k6 will now parse and evaluate thresholds before the execution starts. If a threshold is invalid, as described below, k6 will immediately exit without starting the load test. 
+* k6 will now detect invalid thresholds:
+  * incorrect thresholds expressions:
+    ```javascript
+    export const options = {
+      vus: 5,
+      iterations: 30,
+      thresholds: {
+          http_req_failed: ['rave<0.01'],  // "rave" is not a valid aggregation method
+      },
+    };
+    
+    ``` 
+  * thresholds applying to a non-existing metric
+    ```javascript
+    export const options = {
+      vus: 5,
+      iterations: 30,
+      thresholds: {
+          iDoNotExist: ['p(95)<200'], // The metric we apply the expression to does not exist
+      },
+    };
+    ```
+  * thresholds applying an aggregation method that's unsupported by the metric they apply to
+    ```javascript
+    export const options = {
+      vus: 5,
+      iterations: 30,
+      thresholds: {
+          my_counter: ['p(95)<200'], // Counter metrics do not support the p() aggregation method
+      },
+    };
+    ```
+
+### Support for encrypted TLS private keys
```

Comment:
Why isn't this in "New features"?

Reply:
From my perspective, this is really an improvement over an existing feature. We already supported TLS private keys, just not those that were encrypted with a passphrase. I would leave it there unless you wholeheartedly disagree 😸  (in which case, feel free to un-resolve the conversation.